QuizApp/Data/DataSources/SecureStorage.swift

```diff
@@ -0,0 +1,15 @@
+import KeychainSwift
+
+struct SecureStorageKey {
+
+    static let accessToken = "access_token"
+
+}
+
+class SecureStorage {
+
+    static let shared = KeychainSwift()
+
+    private init() {}
+
+}
```

Comment:
The point of SecureStorage object is to hide the actual implementation of KeychainSwift so we can easily swap it.
Please make a protocol for it and expose all functionality that you need through that protocol/implementation.